### ISSUE TITLE
Wrong duration "by task" in Statistics

### ISSUE THREAD

**Issue by tk0e** [adfinis/timed-frontend#1086]
Hi 

Since a while when filtering for
- Customer
- Project
- Time range from
- Time range to

in the "by Task" tab of the statistics module, the calculation is wrong. It seems as if the "Time range from" value is ignored.

If only "Time range from" is set, but not "Time range to", then the calculation is correct.

I'd appreciate a quick fix or a proposal for a workaround.

Cheers, Thomas

**Comment by derrabauke** [comment]
After a first investigation it seems like a backend issue. The query parameters get set correctly in the frontend. Could you have a look @trowik?

**Comment by trowik** [comment]
Yes, this [issue](https://github.com/adfinis/timed-backend/issues/970) is aleady known.
Looks like the backend can't handle to many filters. Will take a look
